general/src/datascrubbing/snapshots/tests__sanitize_credit_card.snap

```diff
@@ -4,18 +4,18 @@ expression: data
 ---
 {
   "extra": {
-    "foo": "[creditcard]"
+    "foo": "[filtered]"
```

Comment:
Let's make this title-case like in Python

Reply:
Done, thanks!